setup.py

```diff
@@ -11,7 +11,7 @@
 if compiler_type == 'msvc':
     pass
 else:
-    cflags += ['-O0', '-fsanitize=address']
+    cflags += ['-O0', '-Wextra', '-Werror', '-fsanitize=address']
```

Comment:
I'm gonna overwrite this with my changes, but we should remove -fsanitize=address from the defaults